index.js

```diff
@@ -21,18 +21,27 @@ const getGlobal = property => {
 	}
 };
 
-const document = getGlobal('document');
-const Headers = getGlobal('Headers');
-const Response = getGlobal('Response');
-const ReadableStream = getGlobal('ReadableStream');
-const fetch = getGlobal('fetch');
-const AbortController = getGlobal('AbortController');
-const FormData = getGlobal('FormData');
+const globals = [
+	'document',
+	'Headers',
+	'Response',
+	'ReadableStream',
+	'fetch',
+	'AbortController',
+	'FormData'
+].reduce((acc, key) => {
+	Object.defineProperty(acc, key, {
```

Comment:
I would prefer using `Object.defineProperties` (note the `s`) and just a normal `for-of` loop.